src/modelmanager.cpp

```diff
@@ -102,19 +102,24 @@ ModelManager::ModelManager(const std::string& modelCacheDirectory, MetricRegistr
         }
     }
     this->customNodeLibraryManager = std::make_unique<CustomNodeLibraryManager>();
-    if (ovms::Config::instance().cpuExtensionLibraryPath() != "") {
-        SPDLOG_INFO("Loading custom CPU extension from {}", ovms::Config::instance().cpuExtensionLibraryPath());
-        try {
+    try {
+        if (ovms::Config::instance().cpuExtensionLibraryPath() != "") {
+            SPDLOG_INFO("Loading custom CPU extension from {}", ovms::Config::instance().cpuExtensionLibraryPath());
             OV_LOGGER("ov::Core: {}, ieCore->add_extension({})", reinterpret_cast<const void*>(this->ieCore.get()), ovms::Config::instance().cpuExtensionLibraryPath());
             ieCore->add_extension(ovms::Config::instance().cpuExtensionLibraryPath());
             SPDLOG_INFO("Extension added.");
-        } catch (std::exception& ex) {
-            SPDLOG_CRITICAL("Custom CPU extension loading has failed! Reason: {}", ex.what());
-            throw;
-        } catch (...) {
-            SPDLOG_CRITICAL("Custom CPU extension loading has failed with an unknown error!");
-            throw;
-        }
+        } else {
```

Comment:
adding tokenizer extension was not meant to be conditional on empty cpu_extension parameter. The condition might be only on the presence of the extension lib to avoid exception if case it is missing like built without extension

Reply:
Reverted this feature completely. One now needs to use --cpu_extension libopenvino_tokenizers.so. Updated embeddings documentation.